babel.config.js

```diff
@@ -12,16 +12,21 @@ const isUsedByWebpack = Boolean(BABEL_USED_BY_WEBPACK);
 
 const keepCss = Boolean(BABEL_KEEP_CSS);
 
+const fromStorybook = Boolean(FROM_STORYBOOK);
+
 const testFiles = [
   '**/*.test.ts',
   '**/*.test.tsx',
   '**/*.spec.ts',
   '**/*.spec.tsx',
   '**/*.e2e.ts',
   '**/*.e2e.tsx',
-  '**/testing/',
 ];
 
+if (isProduction && !fromStorybook) {
+  testFiles.push('**/testing/', '**/*.stories.tsx');
```

Comment:
```suggestion
  testFiles.push('**/testing/', '**/*.stories.tsx', '**/storybook/');
```
а то сейчас папка `src/storybook` попадает в `dist/`